src/test/java/com/commercetools/emailprocessor/email/EmailProcessorTest.java

```diff
@@ -0,0 +1,232 @@
+package com.commercetools.emailprocessor.email;
+
+
+import com.commercetools.emailprocessor.model.Statistics;
+import com.commercetools.emailprocessor.model.TenantConfiguration;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customobjects.CustomObject;
+import io.sphere.sdk.customobjects.queries.CustomObjectQuery;
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.queries.PagedQueryResult;
+import org.apache.commons.io.IOUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLStreamHandler;
+import java.net.URLStreamHandlerFactory;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+public class EmailProcessorTest {
+
+
+    private EmailProcessor emailProcessor = null;
+    private TenantConfiguration tenantConfiguration = null;
+    private List<CustomObject<JsonNode>> customObjects = null;
```

Comment:
you don't need to initialise with `null`

Reply:
b7b2ba6